db/post_migrate/20200629232159_rename_path_whitelist_to_allowed_paths.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class RenamePathWhitelistToAllowedPaths < ActiveRecord::Migration[6.0]
+  def change
+    rename_column :embeddable_hosts, :path_whitelist, :allowed_paths
```

Comment:
Is this column in a commonly used path? This rename will result in errors until the new code is deployed.